crates/solver/src/settlement_rater.rs

```diff
@@ -30,6 +30,13 @@ use {
     web3::types::AccessList,
 };
 
+// We require from solvers to have a bit more ETH balance then needed
+// at the moment of simulating the transaction, to cover the potential increase
+// of the cost of sending transaction onchain, because of the sudden gas price
+// increase. To simulate this sudden increase of gas price during simulation, we
+// artificially multiply the gas price with this factor.
+const SOLVER_BALANCE_MULTIPLIER: f64 = 5.;
```

Comment:
Isn't 5 a bit high? It could make it hard to execute transactions when the gas is very high.
If we assume that the gas increases by the maximum base fee bump at each block (12.5%) then a multiplier of 5 means that more than 13 blocks (156s) need to pass before needing this multiplier. For comparison, a multiplier of 3 is 9 blocks (108s). (In theory the priority fee could be arbitrarily high, but I've never seen it being very relevant.)